README.md

```diff
@@ -1,21 +1,17 @@
 # open-in-editor (Sourcegraph extension)
 
-Adds a button at the top of files in both Sourcegraph app and code hosts like GitHub (when the Sourcegraph browser extension is installed) that will open the current file in your editor of choice. 
+Adds a button at the top of files in both Sourcegraph app and code hosts like GitHub (when the Sourcegraph browser extension is installed) that will open the current file in Atom. 
 
-![Screenshot](https://user-images.githubusercontent.com/6304497/94799074-dcfd1d80-03e2-11eb-86fc-e55e03380154.png)
+![Screenshot](https://user-images.githubusercontent.com/37420160/96530024-856d0780-1254-11eb-968c-624aff1fd2e5.png)
```

Comment:
**tip**: You can define a light and dark version of the screenshot using `<picture>`: https://sourcegraph.com/github.com/sourcegraph/sourcegraph-code-stats-insights/-/blob/README.md#L6:1